From a0a7f0b550c66535beb71e1bb85a556e128082b4 Mon Sep 17 00:00:00 2001
From: Jianming Qiao <jianming.qiao@bp.renesas.com>
Date: Wed, 11 Sep 2019 02:20:06 +0100
Subject: [PATCH] creating a new Makefile that triggers Makefile.internal to be
 built

Add top level Makefile for Bitbake to pick up that points to our
TensorFlow Lite Arm specific Makefile.

Signed-off-by: Jianming Qiao <jianming.qiao@bp.renesas.com>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..3f73044
--- /dev/null
+++ b/Makefile
@@ -0,0 +1 @@
+include ./tensorflow/lite/tools/make/Makefile.internal
-- 
2.7.4
